opentech/apply/users/forms.py

```diff
@@ -0,0 +1,12 @@
+from django import forms
+from django.utils.translation import ugettext_lazy as _
+
+from wagtail.wagtailusers.forms import UserEditForm, UserCreationForm
+
+
+class CustomUserEditForm(UserEditForm):
+    full_name = forms.CharField(label=_("Full name"), required=False, help_text=_("Leave blank to generate from First and Last name values."))
```

Comment:
I realise the help text is misleading. The full name will not be generated. From first/last. Maybe remove? Will submit a PR at a later point that enforces email as username and makes full_name required